### ISSUE TITLE
Can't write 2 bytes into word peripheral space

### ISSUE THREAD

**Issue by tgtakaoka** [dlbeer/mspdebug#64]
`mw` command can't write 2 bytes data into word peripheral space, 0x100 <= addr < 0x200.

```
$ mspdebug sim
...
(mspdebug) simio add timer TA3
Added new device "TA3" of type "timer".
(mspdebug) md 0x0170 0x10
    00170: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 ff ff |................|
(mspdebug) mw 0x0170 0x11 0x22
(mspdebug) md 0x0170 0x10
    00170: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 ff ff |................|
(mspdebug) mw 0x0170 0x11 0x22 0x33
(mspdebug) md 0x0170 0x10
    00170: 11 22 00 00 00 00 00 00 00 00 00 00 00 00 ff ff |."..............|
(mspdebug) 
```

**Comment by tgtakaoka** [comment]
This issue should be addressed by pull request #62.

**Comment by tgtakaoka** [comment]
Fixed.
